husky_control/CMakeLists.txt

```diff
@@ -7,8 +7,8 @@ catkin_package()
 
 roslaunch_add_file_check(launch)
 
-catkin_install_python(PROGRAMS scripts/stow_ur5 scripts/stow_ur5_left scripts/stow_ur5_right DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
+install(PROGRAMS scripts/stow_ur5 scripts/stow_ur5_left scripts/stow_ur5_right DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Comment:
What's the reason for this change?

Putting it in bin allows it to be run with rosrun, what happens now?